arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -608,69 +1938,290 @@
         },
-        "status": {
-          "externalDocs": {
-            "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
-          },
+        "ProvisioningState": {
           "type": "string",
-          "description": "Status of the namespace. It can be any of these values:1 = Created/Active2 = Creating3 = Suspended4 = Deleting"
+          "description": "Provisioning state of the Namespace."
+        },
+        "Status": {
+          "type": "string",
+          "description": "State of the namespace.",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Created",
+            "Activating",
+            "Enabling",
+            "Active",
+            "Disabling",
+            "Disabled",
+            "SoftDeleting",
+            "SoftDeleted",
+            "Removing",
+            "Removed",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceState",
+            "modelAsString": false
+          }
         },
-        "createdAt": {
+        "CreatedAt": {
           "type": "string",
           "format": "date-time",
           "description": "The time the namespace was created."
         },
-        "serviceBusEndpoint": {
+        "ServiceBusEndpoint": {
           "externalDocs": {
             "url": "http://msdn.microsoft.com/en-us/library/windowsazure/jj873988.aspx"
           },
           "type": "string",
           "description": "Endpoint you can use to perform ServiceBus operations."
         },
-        "subscriptionId": {
+        "SubscriptionId": {
           "type": "string",
           "description": "The Id of the Azure subscription associated with the namespace."
         },
-        "critical": {
+        "Critical": {
           "type": "boolean",
           "description": "Whether or not the namespace is set as Critical."
         },
-        "namespaceType": {
+        "CreateACSNamespace": {
+          "type": "boolean",
+          "description": "Indicates whether to create ACS namespace."
+        },
+        "Enabled": {
+          "type": "boolean",
+          "description": "Specifies whether this instance is enabled."
+        },
+        "NamespaceType": {
           "type": "string",
           "description": "The namespace type.",
           "enum": [
-            "Messaging",
-            "NotificationHub",
-            "EventHub"
+            "Messaging"
           ],
           "x-ms-enum": {
             "name": "NamespaceType",
             "modelAsString": true
           }
         }
       },
-      "x-ms-client-flatten": true,
       "description": "Properties of the Namespace."
     },
+    "ARMSku": {
+      "type": "object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The resource type that this object applies to. Eg: NamespaceType",
+          "enum": [
```

Comment:
Name and Tier both have the same enum values. Looks weird to me. description for Name is confusing as well.